libs/data-access/courses/src/lib/saga.ts

```diff
@@ -263,6 +270,25 @@ export function* removeStudentFromCourse(actions) {
   }
 }
 
+export function* filterCourses(actions) {
+  try {
+    const path = `/api/${PrivateRoutesPath.COURSES}/filter?`;
+    const result = yield call(getByQueryParams, path, actions.payload);
+    if (result) {
```

Comment:
what should happen if result return null?

Reply:
it should return all courses?